_ecosys/liveservices/Telerik-TV.md

```diff
@@ -4,4 +4,4 @@ category: liveservices
 title: "Telerik TV"
 link: "http://tv.telerik.com/services/odata.svc"
 ---
-Telerik’s catalog of libraries, videos, Tags and Series
+Telerik�s catalog of libraries, videos, Tags and Series
```

Comment:
>� [](http://example.com/codeflow?start=7&length=1)

Is this change intended?

Reply:
It was not on purpose and should be fixed now.